src/SFA.DAS.ReferenceData.Types/DTO/EducationOrganisation.cs

```diff
@@ -11,5 +11,6 @@ public class EducationOrganisation
         public string Town { get; set; }
         public string County { get; set; }
         public string PostCode { get; set; }
+        public long URN { get; set; }
```

Comment:
Does it need to be nullable to cater for any existing ones? or is it a brand new data set

Reply:
Yeah, probably a good idea to be on the safe side.  Part of the PR is to recreate the dataset with the URNs in place, but this would blow up with the current data. 👍